src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const allLists = document.querySelectorAll('li');
+
+allLists.forEach((item) => {
+  const nestedList = item.querySelector('ul');
+
+  if (nestedList) {
+    const span = document.createElement('span');
+
+    const text = item.firstChild.textContent.trim();
+
+    span.textContent = text;
+
+    item.firstChild.replaceWith(span);
+
+    span.addEventListener('click', () => {
+      if (nestedList.style.display === 'none') {
+        nestedList.style.display = 'block';
+      } else {
+        nestedList.style.display = 'none';
```

Comment:
Using 'style.display' directly can lead to issues if other styles are applied via CSS. Consider using 'classList.toggle' with a CSS class to manage visibility, which is generally more robust and maintainable.